src/UI/Badge.elm

```diff
@@ -69,27 +75,57 @@ type alias Properties =
     }
 
 
+type alias Options =
+    { icon : Maybe Icon
+    }
+
+
 type BadgeTone
     = ToneGray
     | TonePrimary
     | ToneDanger
     | ToneWarning
     | ToneSuccess
+    | ToneClear
 
 
 type BadgeBrightness
     = Light
     | Dark
 
 
+{-| A variation of the badge with an icon
+-}
+withIcon : Icon -> Badge -> Badge
+withIcon icon (Badge { content, tone, brightness } _) =
+    let
+        properties =
+            Properties content tone brightness
+    in
+    Options (Just icon)
+        |> Badge properties
+
+
+{-| A variation of the badge with outline for light backgrounds.
+
+    Badge.outlineLight "SENT"
+
+-}
+outlineLight : String -> Badge
+outlineLight content =
+    Options Nothing
```

Comment:
I recommend replacing `Options Nothing` with a `defaultOptions` (and then removing the pipe)